internal/pkg/deploy/cloudformation/project.go

```diff
@@ -63,6 +63,70 @@ func (cf CloudFormation) CreateProjectResources(project *archer.Project) error {
 	return nil
 }
 
+// GetProjectResourcesByRegion fetches all the regional resources for a particular region.
+func (cf CloudFormation) GetProjectResourcesByRegion(project *archer.Project, region string) (*archer.ProjectRegionalResources, error) {
+	resources, err := cf.getResourcesForStackInstances(project, &region)
+	if err != nil {
+		return nil, fmt.Errorf("describing project resources: %w", err)
+	}
+	if len(resources) == 0 {
+		return nil, fmt.Errorf("no regional resources for project %s in region %s found", project.Name, region)
+	}
+
+	return resources[0], nil
+}
+
+// GetRegionalProjectResources fetches all the regional resources for a particular project.
+func (cf CloudFormation) GetRegionalProjectResources(project *archer.Project) ([]*archer.ProjectRegionalResources, error) {
+	resources, err := cf.getResourcesForStackInstances(project, nil)
+	if err != nil {
+		return nil, fmt.Errorf("describing project resources: %w", err)
+	}
+	return resources, nil
+}
+
+func (cf CloudFormation) getResourcesForStackInstances(project *archer.Project, region *string) ([]*archer.ProjectRegionalResources, error) {
+	projectConfig := stack.NewProjectStackConfig(project, cf.box)
+	listStackInstancesInput := &cloudformation.ListStackInstancesInput{
```

Comment:
should we paginate or set the `MaxResults` field?

Reply:
Also, I thought there will be only one project stack instance per region in the stackset. But seems like we might have multiple?

Reply:
No you're right - one stack set instance per region. I should add pagination, i'll do that in a follow up pr.